Sources/Harbor/HConfig.swift

```diff
@@ -7,19 +7,9 @@
 
 import Foundation
 
-public struct HConfig {
-    let authProvider: HAuthProviderProtocol?
-    let defaultHeaderParameters: [String: String]?
-    let mTLS: HmTLS?
-    let sslPinningSHA256: String?
-
-    public init(authProvider: HAuthProviderProtocol? = nil, 
-                defaultHeaderParameters: [String : String]? = nil,
-                mTLS: HmTLS? = nil,
-                sslPinningSHA256: String? = nil) {
-        self.authProvider = authProvider
-        self.defaultHeaderParameters = defaultHeaderParameters
-        self.mTLS = mTLS
-        self.sslPinningSHA256 = sslPinningSHA256
-    }
+class HConfig {
+    var authProvider: HAuthProviderProtocol?
+    var defaultHeaderParameters: [String: String]?
+    var mTLS: HmTLS?
+    var sslPinningSHA256: String?
```

Comment:
Why did you change it to a class?

Reply:
I was trying another thing, changed again